### PR TITLE
Remove unused initializers

### PR DESCRIPTION
**Pull request by aapomm**
### Spec
Currently, we have unused initializers in the repository. This PR aims to cleanup the code. Additionally, to parrot the changes in the Pro PR, a welcome thor task is needed.

#### Proposed solution
Remove unused initializers and add a `welcome` thor task to import a welcome package.

### How to test:
1. Acquire a sample welcome package
2. Run the command `thor dradis:setup:welcome WELCOME_PACKAGE`
3. Assert that the templates and the project was imported.

### CODE REVIEW

File: lib/tasks/thorfile.rb [hunk 1]
```diff
@@ -113,6 +113,37 @@ def seed
...
+        Dir.chdir(tmpdir) do
+          system 'tar', "--exclude='._*'", '-xzvf', File.basename(welcome_pack)
+        end
+
+        # Copy templates
+        NoteTemplate.pwd
+        Methodology.pwd
+        FileUtils.cp_r(
+          "#{tmpdir}/welcome/templates/notes/.",
+          Configuration.find_by_name("admin:paths:note_templates").value
```
Comment (etdsoft) [inline lib/tasks/thorfile.rb]: Why don't you stick to the `Configuration.paths_templates_plugins` convention here?

File: lib/tasks/thorfile.rb [hunk 2]
```diff
@@ -113,6 +113,37 @@ def seed
...
+      Dir.mktmpdir do |tmpdir|
+        # Extract welcome package
+        FileUtils.cp welcome_pack, tmpdir
+        Dir.chdir(tmpdir) do
+          system 'tar', "--exclude='._*'", '-xzvf', File.basename(welcome_pack)
+        end
+
+        # Copy templates
+        NoteTemplate.pwd
+        Methodology.pwd
```
Comment (etdsoft) [inline lib/tasks/thorfile.rb]: What's the purpose of L128-L129?

File: lib/tasks/thorfile.rb [hunk 3]
```diff
@@ -113,6 +113,37 @@ def seed
...
+        FileUtils.cp_r(
+          "#{tmpdir}/welcome/templates/notes/.",
+          Configuration.find_by_name("admin:paths:note_templates").value
+        )
+        FileUtils.cp_r(
+          "#{tmpdir}/welcome/templates/methodologies/.",
+          Configuration.find_by_name("admin:paths:templates:methodologies").value
```
Comment (etdsoft) [inline lib/tasks/thorfile.rb]: ditto

File: lib/tasks/thorfile.rb [hunk 4]
```diff
@@ -113,6 +113,37 @@ def seed
       puts "[  DONE  ]"
     end
 
+    desc "welcome WELCOME_PACKAGE", "import files and projects from a specficied welcome package"
```
Comment (etdsoft) [inline lib/tasks/thorfile.rb]: @rachkor questions:

- Is there anything else we'd like to do as part of setting up the Welcome?

- How much of a difference would make to have the Welcome be a `.tar.gz` file? Alternatively we can make the DUP's welcome a `.zip` if we think Zip are more user friendly.

**Comment by brianp** [comment]
Closing legacy pull request. PR has been open to long. If still relevant we'll reopen as an issue but create a new up to date PR.

**Pull request closed without merging**